packages/cdk-graph-plugin-diagram/src/internal/graphviz/entities/nodes.ts

```diff
@@ -143,11 +131,7 @@ export class CfnResourceNode extends ImageNode {
 
     this.attributes.apply(GraphTheme.instance.cfnResourceNode);
 
-    this.resize(
-      GraphTheme.instance.cfnResourceNode.height === ""
-        ? undefined
-        : GraphTheme.instance.cfnResourceNode.height
-    );
+    this.resize();
```

Comment:
These resize inputs enable the theme to override the defaults, are you punting this or have it handled somewhere else? Either is fine for now, given we don't have much in terms of theming atm.

Reply:
Added them back in.